lib/kazan/watcher.ex

```diff
@@ -27,14 +27,12 @@ defmodule Kazan.Watcher do
   is used. i.e. `Kazan.Apis.Core.V1.list_namespace!()`
   rather than `Kazan.Apis.Core.V1.watch_namespace_list!()`
 
-  3. In your client code you receive messages for each `%WatchEvent{}`.
+  3. In your client code you receive messages for each `%Watcher.Event{}`.
     You can pattern match on the object type if you have multiple watchers configured.
     For example, if the client is a `GenServer`
   ```
-    alias Kazan.Models.Apimachinery.Meta.V1.WatchEvent
-
-    # type is "ADDED", "DELETED" or "MODIFIED"
-    def handle_info(%WatchEvent{object: object, type: type}, state) do
+    # type is "ADDED", "DELETED", "MODIFIED" or "GONE"
```

Comment:
Since we're breaking compatibility and no longer using WatchEvent directly, what do you think to converting these strings into atoms?